Dockerfile

```diff
@@ -4,11 +4,13 @@
 
 # BUILD
 FROM node:18-alpine AS builder
+ARG BUILD_VERSION=dev
 ENV NEXT_TELEMETRY_DISABLED=1
 
 WORKDIR /app
 COPY . ./
 RUN yarn install --immutable && \
+    sed -i "s/0.0/${BUILD_VERSION}/" src/version.json && \
```

Comment:
Why not jq?

Reply:
Because it is not included in the alpine node image

Reply:
Hm, yes. Should that have a special meaning in the rendering in HedgeDoc?

Reply:
Just something unique. Like "Frontend version not set during build" :yum: 
I mean more unique than "0.0"